app/javascript/hyacinth_v1/components/digital_objects/metadata/MetadataForm.js

```diff
@@ -59,9 +60,13 @@ function MetadataForm(props) {
   } = digitalObject;
   const [descriptiveMetadata, setDescriptiveMetadata] = useState({});
   const [createDigitalObject, { error: createErrors }] = useMutation(createDigitalObjectMutation);
-  const [updateDescriptiveMetadata, { error: updateErrors }] = useMutation(
+  const [updateDescriptiveMetadata, { data: updateData }] = useMutation(
```

Comment:
You're right to add `data: updateData` here, but we still want to keep and make use of `error: updateErrors` in case there are any top-level, unexpected graphql errors raised.

I've been using the term "userErrors" (which you have below, for errors that appear within the top level "data" object) to indicate that they're errors that the user should be able to fix on their own, but it's still useful to show the top level errors so that users can report those error messages to developers if they come up.